Dockerfile

```diff
@@ -1,11 +1,8 @@
-FROM golang:1.19.7-alpine3.17
+FROM golang:1.20.2-alpine3.17
 
 # add addtional packages needed for the race detector to work
 RUN apk add --update build-base make 
 
-# add a non-root user to run our code as
-RUN adduser --disabled-password --gecos "" appuser
-
```

Comment:
We have this in here for security reasons. I am not sure deleting this is a viable option.

Reply:
Since the containers are ephemeral and run with network disabled, a read-only file system and an execution time timeout, what could be the risk of running as root inside the container?

Reply:
Updated the PR with the fix described here: https://forum.exercism.org/t/tests-failing-with-an-error-occurred/4616/15

This makes the test runner fast, while still using `appuser`

Reply:
💙